app/modules/user/login/login.controller.js

```diff
@@ -27,4 +32,7 @@ const loginUser = async(req, res) => {
 
     }
 }
+myEmitter.on('loginUser', () => {
```

Comment:
Create a folder events outside app. Create a file events.js. Create the event object there. Export it. Require this object to emit events. Also write the .on function inside the new file.